core/cmd/local_client.go

```diff
@@ -491,7 +493,9 @@ func (cli *Client) SetNextNonce(c *clipkg.Context) error {
 	nextNonce := c.Uint64("nextNonce")
 
 	logger.SetLogger(cli.Config.CreateProductionLogger())
-	db, err := gorm.Open(string(orm.DialectPostgres), cli.Config.DatabaseURL())
+	db, err := gorm.Open(gormpostgres.New(gormpostgres.Config{
```

Comment:
We never want to set anything in the gorm.Config?

Reply:
nothing obvious to me, defaults look sane